### ISSUE TITLE
I would like clarification of "…this means no Child Themes. Hooray!"

### ISSUE THREAD

**Issue by stevecj** [eddiemachado/bones#812]
The big benefit of using child themes seems to be the ability to update the base theme (for bug fixes, enhancements, etc.) while retaining customizations. That is a good thing, right?

I assume there is a reason that Bones is intended to be modified directly rather than be used as a parent template, but it would be nice to have an explanation of that on the home page (or linked to from the home page) explaining the reasoning for that and what people _should_ be doing to incorporate updates rather than making child themes.

I'm guessing that the best way to incorporate upstream improvements might be something like…

Start each derived theme as a branch of a clone of the Bones git project, and then rebase the branch onto the latest upstream master of Bones to incorporate the latest.

Is that the idea (or at least a perfectly good idea)?

**Comment by KTS915** [comment]
No, the idea is to create your own custom theme. And, because each such theme is, indeed, custom, there can then be no parent theme and no child themes. Hooray!
